scripts/dap-ami-bootstrap.sh

```diff
@@ -1,38 +1,42 @@
 #!/bin/bash
 set -e
 
-if ! docker images | grep conjur; then
-  echo "Loading Conjur appliance image into Docker"
-  gzip -df /home/core/conjur-appliance.tar.gz # Unzip the tar.gz into a tar
-  docker load -i /home/core/conjur-appliance.tar
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
 fi
 
 image_id=$(docker images -q)
-container_name='conjur-appliance'
+container_name='dap-appliance'
```

Comment:
Names are hot-button items, but what do we think about `cyberark-dap` as name? (@alexkalish, I'd love your thoughts as well.

Reply:
Good point, Jason.  We need to ask PM.  They have actually been moving away from using the term appliance, but I also know that it's ubiquitous.  Could I get a quick TL;DR on how/where names are changing in this PR so that I can talk to PM about it?  Thanks!

Reply:
As a middle-road option I've just removed "appliance".